### DISCUSSION TITLE
Split boundary processed by XYDelaunayGenerator

### DISCUSSION THREAD

**Discussion by lipxisi** [idaholab/moose#27780] [Q&A Meshing] [answered]
hello

The grid processed by XYDelaunayGenerator will merge the outer boundary of the geometry. How can I restore the previous boundary。I  need the previous rmax, dmin and dmax boundaries.

  [annular]
    type = AnnularMeshGenerator
    nr = 15
    rmin = 0
    rmax = 0.005
    nt = 24
    dmin = 0
    dmax = 90
    equal_area = true
  []
  [triangulates]
    type = XYDelaunayGenerator
    boundary = annular
  []

Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

If there is not a parameter in the triangulator to do this, then simply re add that boundary with ParsedGenerateSideset

Circular boundaries are each to get an analytical expression for
